src/xenia/kernel/xfile.h

```diff
@@ -76,6 +76,7 @@ class X_FILE_DIRECTORY_INFORMATION {
 class XFile : public XObject {
  public:
   static const XObject::Type kObjectType = XObject::Type::File;
+  static constexpr size_t kMaxPathLength = 255;
```

Comment:
It's not the global maximum path length, seems to be specific to LoadedImageName until we have evidence to the contrary.